### ISSUE TITLE
Questions about reflectance values

### ISSUE THREAD

**Issue by seanhardison1** [GlobalHydrologyLab/AquaSat#44]
Thanks for putting together such an incredible data set. I've been looking through the data currently posted on [figshare](https://figshare.com/articles/AquaSat/8139383/2), and was wondering if you'd be able to clarify some issues I'm running into.

* There appear to be negative reflectance values for some observations. Any idea why that might be?
* Many reflectance values are equal to 20000 but with corresponding standard deviations of 0. Are these errors?
* The L8 product [guide](https://www.usgs.gov/media/files/land-surface-reflectance-code-lasrc-product-guide) isn't super helpful in describing what the QA pixel values mean in plain english. Do you have any further guidance on which values should be kept, or maybe a more helpful source for noobs?

**Comment by matthewross07** [comment]
Hi Sean, 

Sorry for the long delay, I turned notifications off for this repo. These are indeed pure errors, that we probably should have just directly filtered out prior to publishing. We will do so in the future. Negative values and 20000 are not right . QA bands are indeed confusing! But the main goal is to highlight clouds or other issues with the data. Hope this helps!

**Comment by seanhardison1** [comment]
Thanks so much! That clarifies things.
